.github/workflows/test.yml

```diff
@@ -46,21 +48,52 @@ jobs:
   build-docker:
     runs-on: ubuntu-latest
     container:
-      image: ubuntu:focal
+      image: osrf/ros2:nightly
     steps:
-      - name: Setup ROS2
-        uses: ros-tooling/setup-ros@2c05014
+      - name: prep
+        run: |
+          apt-get -qq update
+          apt-get -qq upgrade -y
+          apt-get -qq install -y curl libasio-dev libtinyxml2-dev
+          colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
```

Comment:
Why is this different than above?

Reply:
Setup-Ros runs rosdep init, which fails in the osrf container since its already initialized

Reply:
The above was copied from SROS, this one was written manually. I'll change this with pinning to *A* commit, just the current master instead

Reply:
The specific commit above was the most recent edit to the file on master. I'll change this to match.